### PR TITLE
add strikethrough prop

### PR DESCRIPTION
**Pull request by ellell**
Type: Minor
Review: @kesla @iefserge

### CODE REVIEW

File: lib/text-formattings.js [hunk 1]
```diff
@@ -57,6 +57,18 @@ const defaultTextFormattings = [
...
+  {
+    property: 'strikethrough',
+    get: elm => {
+      if (!elm.tagName) {
+        return false;
+      }
+
+      const tagName = elm.tagName.toLowerCase();
+      return tagName === 's' ||
+        tagName === 'strike';
```
Comment (kesla) [inline lib/text-formattings.js]: should we identify `<del>` as strikethrough as well?

Comment (ellell) [inline lib/text-formattings.js]: yup, I'll add!

**Comment by danmakenoise** [comment]
The only issue I take with this, and is not related to this PR specifically is that there is a TON of repetition in this JSON files we require. Is there a reason these have to be pure JSON files, and not scripts that export the resulting object you want? I just saw that you had to add the same thing in a bunch of different places because of this.

Other than that, everything looks fine.

**Comment by ellell** [comment]
@danmakenoise Yes there's a lot of repetition in the tests, I tried finding a better way to structure it for this PR but couldn't really think of anything where readability wouldn't suffer.

**Comment by kesla** [comment]
@danmakenoise I don't really see a big problem w repeating in tests, tbh.

Having a script exporting the resulting object means we write more code. More code that can have issues. So I somewhat prefer then to repeat things pure in JSON...

**Comment by danmakenoise** [comment]
@kesla okay!

**Pull request merged**